src/app/components/List/EditionsList.jsx

```diff
@@ -3,21 +3,33 @@ import PropTypes from 'prop-types';
 import * as DS from '@nypl/design-system-react-components';
 import EditionCard from '../Card/EditionCard';
 
-const getAllEditionsData = (editions, origin, eReaderUrl, referrer) => editions.map(
-  edition => EditionCard.getEditionData(edition, origin, eReaderUrl, referrer),
-);
 
 class EditionsList extends React.Component {
   constructor(props) {
     super(props);
     this.props = props;
-    this.state = { loaded: false };
+
+    this.getAllEditionsData = this.getAllEditionsData.bind(this);
+    this.state = {
+      loaded: false,
+    };
   }
 
   componentDidMount() {
     this.setState({ loaded: true });
   }
 
+  getAllEditionsData(work, editions, origin, eReaderUrl, referrer) {
+    /* eslint-disable-next-line no-underscore-dangle */
+    return editions.map(
+      (edition) => {
+        const showRequestButton = this.props.getRequestEditionButton(edition);
+        const blah = EditionCard.getEditionData(edition, origin, eReaderUrl, referrer, showRequestButton);
```

Comment:
Can probably just return this directly (or replace this excellent variable name haha)